pxr/usd/usdGeom/mesh.h

```diff
@@ -54,28 +54,69 @@ class SdfAssetPath;
 
 /// \class UsdGeomMesh
 ///
-/// Encodes a mesh surface whose definition and feature-set
-/// will converge with that of OpenSubdiv, http://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html.
-/// Certain interpolation ("tag") parameters are not yet supported.
+/// Encodes a mesh with optional subdivision properties and features.
 /// 
+/// Meshes are conceptually defined in terms of vertices, edges and faces.
+/// As a point-based primitive, the terms "vertex" and "point" are synonymous,
+/// as are associated terms such as "point index" and "vertex index", etc.
+/// The encoding defines the mesh topology completely in terms of a set of
+/// faces and their defining face-vertices (see _faceVertexCounts_ and
+/// _faceVertexIndices_).  No boundary representation or additional
```

Comment:
Is there a way to make these link to the attribute docs below? (Just curious!)

Reply:
Do we really want to?  Originally I did not include them, but I saw forward references to the "subdivisionScheme" attribute in the following paragraph and so followed the convention and emphasis.  Should we similarly link "subdivisionScheme"?

Reply:
This has now been addressed merging Spiff's suggestions in the latest commit.